index.ts

```diff
@@ -12,6 +12,18 @@ type Attributes = JSX.IntrinsicElements['div'];
 type DocumentFragmentConstructor = typeof DocumentFragment;
 type ElementFunction = () => HTMLElement | SVGElement;
 
+declare global {
```

Comment:
`global` might not be needed, we don't use it here: https://github.com/sindresorhus/refined-github/blob/62b69a571f9e43c07b7e18911226e4f6467b9829/source/globals.d.ts#L47-L48

Reply:
im declaring it in `index.ts` not in `global.d.ts` so the `global` keyword is needed, maybe i should move it to `global.d.ts`.